File: src/Test/ArrayOpsFFICheatcode.go
package Test_ArrayOpsFFICheatcode


func RunArrayOpsFFICheatcode(limit int) int {
	n := int(limit)
	sum := 0
	for i := 1; i <= n; i++ {
		if i%2 == 0 {
			sum += i
		}
	}
	return (sum)
}

File: src/Test/ArrayOpsFFICheatcode.php
<?php
$exports['runArrayOpsFFICheatcode'] = function($limit) {
    $end = (int)$limit;
    $step = $end >= 1 ? 1 : -1;
    $sum = 0;
    for ($value = 1; ; $value += $step) {
        if ($value % 2 === 0) $sum += $value;
        if ($value === $end) break;
    }
    return $sum;
};
return $exports;
